test/integration/main.bats

```diff
@@ -3,7 +3,7 @@
 load helpers
 
 @test "test that create doesn't accept any arugments" {
-        run swarm create derpderpderp
```

Comment:
Not sure what changed here (spaces / tabs)?

Also, not entirely sure if we need this test at all - it sure doesn't hurt.